http/src/test/resources/diagnostics/lightblue-healthy-response.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Any reason this one still has the abbreviated response?  Is it just to test the minimal values?

Reply:
I am only interested in "healthy" for any lightblue diagnostics element. So, just kept it minimal here.